Makefile

```diff
@@ -9,14 +19,20 @@ aws-deps:
 azure-deps: jq-dep
 	npm install -g azure-cli
 
-jq-dep:
+jq-dep: $(GOPATH)/bin/jq
+
+$(GOPATH)/bin/jq:
 	@echo "Downloading jq..."
 	mkdir -p $(GOPATH)/bin
 	wget "https://github.com/stedolan/jq/releases/download/jq-1.5/jq-linux64" -O $(GOPATH)/bin/jq
 	chmod "+x" $(GOPATH)/bin/jq
```

Comment:
ok!

You can read more about the travis container based infra here: https://docs.travis-ci.com/user/installing-dependencies/#Installing-Packages-on-Container-Based-Infrastructure